Dockerfile

```diff
@@ -8,8 +8,8 @@
 #   docker run -it --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -v `pwd`:/pwndbg pwndbg bash
 #
 
-ARG image=ubuntu:20.04
-FROM $image
+ARG CODE_NAME=jammy
+FROM mcr.microsoft.com/devcontainers/base:${CODE_NAME}
```

Comment:
This dockerfile is used to test pwndbg on various distros and this change breaks it.

You can see it e.g. here: https://github.com/pwndbg/pwndbg/actions/runs/7274796295/job/19821439762?pr=1964 where a build that should run debian11 now runs off the microsoft image due to this change